pkg/e2e/fixtures/dotenv/raw.yaml

```diff
@@ -0,0 +1,7 @@
+services:
+  test:
+    image: alpine
+    command: sh -c "echo $$TEST_VAR"
+    env_file:
+      - path: .env.raw
+        format: raw # parse without interpolation
```

Comment:
Do you think there can be other formats? I was wondering if it could be a boolean instead `raw:true` instead

Reply:
my goal is to allow reusing docker/cli parser, which this repository can't refer to.
Also, while we don't have another format yet, I could imaging one might want to use java-style properties files, or maybe xml 🧌